src/uct/tcp/tcp_cm.c

```diff
@@ -208,7 +208,7 @@ unsigned uct_tcp_cm_conn_progress(uct_tcp_ep_t *ep)
     }
 
     uct_tcp_cm_change_conn_state(ep, UCT_TCP_EP_CONN_STATE_WAITING_ACK);
-    uct_tcp_ep_mod_events(ep, EPOLLIN, EPOLLOUT);
+    uct_tcp_ep_mod_events(ep, UCS_EVENT_SET_EVREAD, UCS_EVENT_SET_EVWRITE);
```

Comment:
pls, change type of `uct_tcp_ep_t::events` from `uint32_t` to `int`

Reply:
Does this change is the same meaning as https://github.com/openucx/ucx/pull/3698#discussion_r293509403?

Reply:
yes, but this is for `uct_tcp_ep_t` structure that has `events` field

Reply:
Changed `uct_tcp_ep` too

```diff
--- a/src/uct/tcp/tcp.h
+++ b/src/uct/tcp/tcp.h
@@ -98,7 +98,7 @@ struct uct_tcp_ep {
     uint8_t                       ctx_caps;    /* Which contexts are supported */
     int                           fd;          /* Socket file descriptor */
     uct_tcp_ep_conn_state_t       conn_state;  /* State of connection with peer */
-    uint32_t                      events;      /* Current notifications */
+    int                           events;      /* Current notifications */
     uct_tcp_ep_ctx_t              tx;          /* TX resources */
     uct_tcp_ep_ctx_t              rx;          /* RX resources */
     struct sockaddr_in            peer_addr;   /* Remote iface addr */
```